maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -271,26 +271,26 @@ class AndroidDriver(
         val deviceInfo = deviceInfo()
         when (swipeDirection) {
             SwipeDirection.UP -> {
-                val startX = deviceInfo.widthPixels / 2
-                val startY = deviceInfo.heightPixels
-                val endY = deviceInfo.heightPixels / 2
+                val startX = deviceInfo.widthGrid / 2
+                val startY = deviceInfo.heightGrid / 1
```

Comment:
Hm, doesn't this mean it will swipe from the very bottom of the screen still? (`deviceInfo.heightGrid / 1 == deviceInfo.heightGrid `)